microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,30 +33,43 @@ The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatic
 <!-- ====================================================================== -->
 ## Before you begin
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center.
+To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials.
 
 > [!NOTE]
-> The **Publish APIs** UI is not yet present at Partner Center.
+> The **Publish API** page is only present at Partner Center for participants of the private preview.
 
 1. Visit Microsoft Partner Center and login to the account from which you have already published an add-on.
 
-1. Under the **Microsoft Edge** program, select **Publish APIs**.
+1. Under the **Microsoft Edge** program, select **Publish API**.
 
-1. In the **Publish APIs** page, select the **Create API credentials** button to generate the API credentials.  This step may take a few minutes.  After the APIs are enabled, the **Client ID**, **Client Secret**, and **Auth Token URL** are displayed on this page.
+1. In the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes.
 
-1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**.  You'll use them in the next step to get an access token.
+   > [!div class="mx-imgBorder"]
+   > ![The 'Publish API' page at Partner Center, with a 'Create API credentials' button.](https://user-images.githubusercontent.com/11265822/150777593-42cb9d37-0c1f-4ef4-9de6-9c4bc1166b96.png)
+
+   <!-- make window smaller before capture.  make fresh capture.  save .png in present repo as ../media/create-api-credentials-button.png, replacing path in parens -->
+
+   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client Secret**, and **Auth Token URL** are now displayed on the Publish APIs page.
+
+   <!-- possibly add a screenshot here, with fake data with the correct number of characters, or value strings blurred out or erased -->
```

Comment:
Possibly add a screenshot here, with fake data with the correct number of characters, or value strings blurred out or erased.